spec/lib/tasks/pay_now_click_tracking_spec.rb

```diff
@@ -28,25 +28,51 @@
       )
     end
 
-    before do
-      payment_transaction1.reload
-      payment_transaction2.reload
+    context 'carrier for dc' do
```

Comment:
there should be no mention of clients in spec files -- please refactor to generalize these specs